src/main/scala/viper/silver/plugin/standard/inline/InlineRewrite.scala

```diff
@@ -3,14 +3,22 @@ package viper.silver.plugin.standard.inline
 import scala.collection.mutable
 import viper.silver.ast._
 import viper.silver.ast.utility.ViperStrategy
-import viper.silver.ast.utility.rewriter.Traverse
+import viper.silver.ast.utility.rewriter.{ContextC, Traverse}
 
 trait InlineRewrite extends PredicateExpansion {
 
+  private type Context = ContextC[Node, Set[String]]
+
   def inlinePredicates(method: Method, program: Program, cond: String => Boolean): Method = {
     val expandedPres = method.pres.map(expandPredicates(_, method, program, cond))
     val expandedPosts = method.posts.map(expandPredicates(_, method, program, cond))
+    val rewrittenBody = method.body.map { ss =>
+      ss.copy(
+        ss = ss.ss.map(expandInhaleStatement(_, method, program, cond))
```

Comment:
`ss.ss.map(....)` is some really cursed code. Makes me want to rewrite the case class

Reply:
haha Viper go ssssssss